codegen/src/main/java/io/improbable/keanu/codegen/python/VertexProcessor.java

```diff
@@ -19,18 +19,24 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
 
 class VertexProcessor {
 
     final private static String TEMPLATE_FILE = "generated.py.ftl";
     final private static String GENERATED_FILE = "generated.py";
+    final private static String TEMPLATE_INIT_FILE = "__init__.py.ftl";
+    final private static String GENERATED_INIT_FILE = "__init__.py";
 
     static void process(String generatedDir) throws IOException {
         Map<String, Object> dataModel = buildDataModel();
         Template fileTemplate = TemplateProcessor.getFileTemplate(TEMPLATE_FILE);
```

Comment:
Can we rename these to something like `generatedFileTemplate` now that we also have an init file template?